lib/remote/configpackageutility.cpp

```diff
@@ -367,7 +378,17 @@ bool ConfigPackageUtility::ContainsDotDot(const String& path)
 	return false;
 }
 
-bool ConfigPackageUtility::ValidateName(const String& name)
+bool ConfigPackageUtility::ValidatePackageName(const String& packageName)
+{
+	return ValidateFreshName(packageName) || PackageExists(packageName);
+}
+
+bool ConfigPackageUtility::ValidateStageName(const String& packageName, const String& stageName)
+{
+	return ValidateFreshName(stageName) || StageExists(packageName, stageName);
```

Comment:
Is this required for stages? Shouldn't a stage name always be a UUID chosen by Icinga and thus also be valid using the new validation function? According to https://icinga.com/docs/icinga-2/latest/doc/12-icinga2-api/#create-a-stage-upload-configuration, there's no option to influence the stage name.